elements/pl-card/pl-card.py

```diff
@@ -0,0 +1,56 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+
+
+HEADER_DEFAULT = ""
+TITLE_DEFAULT = ""
+SUBTITLE_DEFAULT = ""
+FOOTER_DEFAULT = ""
+IMG_TOP_SRC_DEFAULT = ""
+IMG_BOTTOM_SRC_DEFAULT = ""
+WIDTH_DEFAULT = "auto"
+
+
+def prepare(element_html: str, data: pl.QuestionData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    required_attribs = []
+    optional_attribs = [
+        "header",
+        "title",
+        "subtitle",
+        "footer",
+        "img-top-src",
+        "img-bottom-src",
+        "width",
+    ]
+    pl.check_attribs(element, required_attribs, optional_attribs)
+
+
+def render(element_html: str, data: pl.QuestionData) -> str:
+    element = lxml.html.fragment_fromstring(element_html)
+
+    header = pl.get_string_attrib(element, "header", HEADER_DEFAULT)
+    title = pl.get_string_attrib(element, "title", TITLE_DEFAULT)
+    subtitle = pl.get_string_attrib(element, "subtitle", SUBTITLE_DEFAULT)
+    footer = pl.get_string_attrib(element, "footer", FOOTER_DEFAULT)
+    img_top_src = pl.get_string_attrib(element, "img-top-src", IMG_TOP_SRC_DEFAULT)
+    img_bottom_src = pl.get_string_attrib(element, "img-bottom-src", IMG_BOTTOM_SRC_DEFAULT)
+    width = pl.get_string_attrib(element, "width", WIDTH_DEFAULT)
+
+    content = pl.inner_html(element)
+
+    html_params = {
```

Comment:
I think black is complaining about line 43? @eliotwrobson any thoughts as to why?

```
./elements/pl-card/pl-card.py:3[8](https://github.com/PrairieLearn/PrairieLearn/actions/runs/3973531154/jobs/6812236198#step:17:9):43: BLK100 Black would make changes.
```

Reply:
I ran `lint-py` manually and it seems the issue was up at the `img_bottom_src` line: 8c5b86e246ee6925134949cc7f8fc822e71671be

Reply:
You have to run `make format`

Reply:
Oops yes, `format` is one that fixed 8c5b86e246ee6925134949cc7f8fc822e71671be